tests/subsys/logging/log_syst/prj.conf

```diff
@@ -0,0 +1,21 @@
+CONFIG_LOG=y
+CONFIG_ASSERT=n
+CONFIG_ZTEST=y
+CONFIG_LOG_BACKEND_MOCK=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_SOC_LOG_LEVEL_OFF=y
+CONFIG_ARCH_LOG_LEVEL_OFF=y
+CONFIG_KERNEL_LOG_LEVEL_OFF=y
+CONFIG_LOG_RUNTIME_FILTERING=n
+CONFIG_MAIN_THREAD_PRIORITY=5
+CONFIG_LOG_FUNC_NAME_PREFIX_DBG=n
+
```

Comment:
Same thing, do you need to turn off function name prefix for this test ?

Reply:
The CONFIG_LOG_FUNC_NAME_PREFIX_DBG is needed as it results in modification of output str which is used for validation here.